src/backend/catalog/catalog.c

```diff
@@ -362,9 +362,20 @@ IsExtAuxNamespace(Oid namespaceId)
  *		system objects only.  As of 8.0, this was only true for
  *		schema and tablespace names.  With 9.6, this is also true
  *		for roles.
+ */
+bool
+IsReservedName(const char *name)
+{
+	/* ugly coding for speed */
+	return name[0] == 'p' && name[1] == 'g' && name[2] == '_';
```

Comment:
`pg_ ` as prefix ? but comments `IsReservedGpName True iff name starts with the gp_ prefix. ` ?